x/leverage/keeper/keeper.go

```diff
@@ -9,7 +9,6 @@ import (
 	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
 	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
 	"github.com/tendermint/tendermint/libs/log"
-
 	"github.com/umee-network/umee/x/leverage/types"
```

Comment:
```suggestion

	"github.com/umee-network/umee/x/leverage/types"
```
apply import conventions

Reply:
done at